source/_integrations/group.markdown

```diff
@@ -128,6 +145,19 @@ binary_sensor:
       - binary_sensor.door_right_contact
 ```
 
+Example YAML configuration of a climate group:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: group
+    name: "Downstairs Climate"
+    entities:
+      - climate.hall
+      - climate.living_room
+    temperature_unit: '°C' # optional '°F' or '°C'
```

Comment:
Why would this have a temperature unit?
None of our groups provide additional configuration.

Reply:
Is there a possibility to check which temperature unit a climate entity has? 

The unit defaults to the home-assistant unit, but if my climate entities have a different unit, I did not find a way to look that up via state or attributes.

Maybe also good to add a note:

```suggestion
    temperature_unit: '°C' # optional '°F' or '°C', defaults to the unit set in the home-assistant config 
```